vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -188,7 +196,8 @@ def pre_build(self):
         # filter targets at least one figure that uses dynamic data source. Note that min or max = 0 are Falsey values
         # but should still count as manually set.
         if (
-            not getattr(self.selector, "options", [])
+            isinstance(self.selector, (SELECTORS["categorical"], SELECTORS["numerical"], SELECTORS["temporal"]))
```

Comment:
Maybe clearer and I think should do the same thing?

```suggestion
            not isinstance(self.selector, SELECTORS["boolean"])
```